agentMET4FOF/metrological_agents.py

```diff
@@ -146,5 +193,17 @@ def custom_plot_function(self, data, sender_agent, **kwargs):
                 trace = go.Scatter()
         else:
             trace = go.Scatter()
-
+        # print(data.shape)
+        # t=np.linspace(0,1,50)
+        # v=np.sin(2*np.pi*t)
+        # ut=.001*np.ones_like(t)
+        # uv=.005*np.ones_like(v)
+        # trace = go.Scatter(
+        #     x=t,
+        #     y=v,
+        #     error_x=dict(type="data", array=ut, visible=True),
+        #     error_y=dict(type="data", array=uv, visible=True),
+        #     mode="lines",
+        #     name=f"blah blah",
+        # )
```

Comment:
again if the unused comments can be removed from the final code, thatd be great. thanks

Reply:
Done. There were a few commented lines in the original code in the MetrologicalAgents class. @mgrub Can these be removed as well? Lines 11-15 and 19-23.